lua/docs-view.lua

```diff
@@ -44,6 +44,18 @@ local function toggle()
       callback = function()
         if win and vim.api.nvim_win_is_valid(win) then
           local l, c = unpack(vim.api.nvim_win_get_cursor(0))
+          local clients = vim.lsp.buf_get_clients(0)
```

Comment:
Seems like `vim.lsp.buf_get_clients` is deprecated now and we should use `vim.lsp.get_active_clients`

Reply:
I tried to see when was this API introduced but I couldn't find it. I think this plugin needs to at least support 0.7 so I guess we could check nvim version and use `vim.lsp.get_active_clients()` on 0.8 and higher

Reply:
Done.